py/server/tests/test_parquet.py

```diff
@@ -103,14 +111,14 @@ def test_crd_with_instructions(self):
             self.assertTrue(os.path.exists(file_location))
             shutil.rmtree(base_dir)
 
-        with self.subTest(msg="write_tables(Table[], destinations, col_definitions, "):
-            batch_write([table, table], [file_location, file_location2], col_definitions,
+        with self.subTest(msg="write_tables(Table[], destinations, table_definition, col_instructions"):
+            batch_write([table, table], [file_location, file_location2], table_definition,
```

Comment:
Do we still test the case where `table_definition` is a `List[Columns]`?

Reply:
There is a test for that case under `test_write_partitioned_data` around line 649 where we pass a list of columns while writing.